src/magic_folder/api_cli.py

```diff
@@ -220,62 +234,17 @@ def monitor(options):
     return Deferred()
 
 
-class MagicFolderApiCommand(usage.Options):
+class MagicFolderApiCommand(BaseOptions):
```

Comment:
The logic I removed here was almost identical to the main cli command one (with slightly different names). By sharing the logic here, it makes it easier to share the logic in `magic_folder.test.cli.common` for running commands in process. (Particularly the logic around setting config and http_clients)